org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/jsts/ui/preferences/JSTSPreferenceServerConstants.java

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************************

```

Comment:
Why not to merge this class into https://github.com/eclipse-wildwebdeveloper/wildwebdeveloper/blob/master/org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/jsts/ui/preferences/typescript/TypeScriptPreferenceServerConstants.java ?

Reply:
Are you also suggesting to move the preference to the TypeScript subsection in the UI? My rationale was that this preference effectively impacted the language server as a whole, so I put it somewhere top level.

Reply:
No. I think you're right. Let's live it as it is - to be a common setting for the language server.